src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmManagementGroup.md

```diff
@@ -0,0 +1,235 @@
+---
+external help file: Microsoft.Azure.Commands.ManagementGroups.dll-Help.xml
+Module Name: AzureRM.ManagementGroups
+online version: https://docs.microsoft.com/en-us/powershell/module/get-azurermmanagementgroup/
```

Comment:
This should be: https://docs.microsoft.com/en-us/powershell/module/azurerm.managementgroups/get-azurermmanagementgroup (with the module name).  Apply everywhere.